daemon/main.go

```diff
@@ -262,6 +262,10 @@ func init() {
 		"debug", "D", false, "Enable debugging mode")
 	flags.StringVarP(&config.Device,
 		"device", "d", "undefined", "Device facing cluster/external network for direct L3 (non-overlay mode)")
+	flags.StringVarP(&config.DevicePreFilter,
+		"prefilter-device", "", "undefined", "Device facing external network for XDP prefiltering")
+	flags.StringVarP(&config.ModePreFilter,
+		"prefilter-mode", "", "native", "Prefilter mode { native | generic } (default: native)")
```

Comment:
You want to use the const here instead of `"native"`

Reply:
Yep, makes sense, will do.

Reply:
Both fixed.